### PR TITLE
stat: add population (biased) variants of Variance, StdDev, MeanVariance and MeanStdDev

### PR DESCRIPTION
**Pull request by romanwerpachowski**
Please take a look.

### CODE REVIEW

File: stat/stat.go [hunk 1]
```diff
@@ -1261,6 +1261,54 @@ func Variance(x, weights []float64) float64 {
...
+}
+
+// PopulationMeanVariance computes the sample mean and biased variance (also known as
+// "population variance"), where the mean and variance are
+//  \sum_i w_i * x_i / (sum_i w_i)
+//  \sum_i w_i (x_i - mean)^2 / (sum_i w_i)
+// respectively.
+// If weights is nil then all of the weights are 1. If weights is not nil, then
+// len(x) must equal len(weights).
+func PopulationMeanVariance(x, weights []float64) (mean, variance float64) {
```
Comment (kortschak) [inline stat/stat.go:1279]: `PopMeanVariance`? I'm don't have a strong opinion either way.

(applied throughout if adopted)

Comment (romanwerpachowski) [reply stat/stat.go]: Done.

**Review by kortschak — DISMISSED** [review]
Thanks

**Comment by kortschak** [comment]
Good to see you back.

**Comment by rilwen** [comment]
I had to amend the last commit message. Can you reapprove?

**Comment by romanwerpachowski** [comment]
Good to be back! I had to amend the last commit message. Can you reapprove, please?

**Review by kortschak — APPROVED** [review]
This will all get squashed into a single commit yeah?

**Comment by romanwerpachowski** [comment]
Will do.

**Pull request merged**